src/06_tuples.py

```diff
@@ -17,28 +17,33 @@
 
 import math
 
+
 def dist(a, b):
     """Compute the distance between two x,y points."""
     x0, y0 = a  # Destructuring assignment
     x1, y1 = b
-    
+
     return math.sqrt((x1 - x0)**2 + (y1 - y0)**2)
 
+
 a = (2, 7)   # <-- x,y coordinates stored in tuples
 b = (-14, 72)
 
 # Prints "Distance is 66.94"
 print("Distance is: {:.2f}".format(dist(a, b)))
 
 
-
 # Write a function `print_tuple` that prints all the values in a tuple
 
 # YOUR CODE HERE
+def print_tuple(items):
+    for item in items:
+        print(item)
+
```

Comment:
Clean and readable! I appreciate you using `item` as your traversal variable as opposed to `i, for example.